src/getCoinCombination.test.js

```diff
@@ -7,4 +7,28 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should be return '[1, 1, 0, 0]' when argument '6'`, () => {
+    const result = getCoinCombination(6);
+
+    expect(result).toEqual([1, 1, 0, 0]);
+  });
+
+  it(`should be return '[0, 0, 0, 2]' when argument '50'`, () => {
+    const result = getCoinCombination(50);
+
+    expect(result).toEqual([0, 0, 0, 2]);
+  });
+
+  it(`should be return '[2, 1, 1, 0]' when argument '17'`, () => {
+    const result = getCoinCombination(17);
+
+    expect(result).toEqual([2, 1, 1, 0]);
+  });
+
+  it(`should be return '[, 0, 0, 0]' when argument '1'`, () => {
```

Comment:
The test description should be corrected to 'should return [1, 0, 0, 0] when argument is 1' for clarity and grammatical correctness. Additionally, there is a typo in the expected array in the description: '[, 0, 0, 0]' should be '[1, 0, 0, 0]'.